scripts/visualize/visualize_map.R

```diff
@@ -8,6 +8,16 @@ visualize.states_svg <- function(viz){
   shifts <- state.map$shifted.states
   centroids <- state.map$state.centroids
   state.name <- as.character(row.names(states)[states@plotOrder])
+  
+  # create hovertext, just capitalized state names for now
+  capwords <- function(s) {
+    capword <- function(w) {
+      paste(ifelse(w == 'of', w, paste0(toupper(substring(w, 1, 1)), substring(w, 2))), collapse=" ")
+    }
+    sapply(strsplit(s, split = " "), capword, USE.NAMES = FALSE)
+  }
```

Comment:
putting this capitalization function here works for the initial tooltips. i think as we get toward actually changing the tooltips for each category/year switch, we may want to put this into the function that creates js/scaleFactors.json - i.e., put the pre-capitalized words in there instead of sticking with lowercase until tooltip time.